.github/ISSUE_TEMPLATE/feature_request.yml

```diff
@@ -0,0 +1,20 @@
+name: Feature Request
+description: Have an idea to improve the Homebrewery? Let us know!
+labels: ["feature request"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        We'd love to hear your idea!  Please be sure to search the current Issues for any duplicate requests.
```

Comment:
Is it possible to make `current Issues` a link to the issues page?

Reply:
I think so...it seems like regular markdown should work, so i've done that.  I'm not sure how to test this PR without it being on the live repo, so might need to push it live and see what happens....

Reply:
Right, I'm not seeing any way to preview what these will look like either. Even in their demo video they're like "so lets commit this now so we can see what it looks like." It does seem to use markdown though, so you're right I think this should work...